### PR TITLE
Fix versioning issues

### PR DESCRIPTION
**Pull request by sebasdeldi**
## Summary
* Runnable basic project structure
* Database configuration (Postgres)
* Only API related files included (no assets or views)

## Trello Card
https://trello.com/c/02fBYHK5/14-crear-aplicaci%C3%B3n-rails-wbooksapi

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# available as an environment variable. For example:
+#
+#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
+#
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
```
Comment (nicozar95) [inline config/database.yml:83]: As this is the production config we should not push the wbooks-api DB username and password directly, we could use an `.env` file with all the sensitive data so we change the production DB values to something like this:
``` 
url: <%= ENV['DATABASE_URL'] %>
```

Also we can set the username and password as default by doint the following:
```
default: &default
  . . .
  username: <%= ENV.fetch('DB_USERNAME', 'wbooks-api') %>
  password: <%= ENV.fetch('DB_PASSWORD', 'wbooks-api') %>
  . . .
``` 

Remember to add the `.env` file on the `.gitignore`

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Great work! some minor changes

**Comment by nicozar95** [comment]
Remember to add @Gmarn as a reviewer too and assign him also

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  <<: *default
+  database: wbooks-api_production
+  username: <%= ENV.fetch('DB_USERNAME') %>
```
Comment (nicozar95) [inline config/database.yml]: Move the username and password to the default tag in line 17

**Review by nicozar95 — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 2 above)
Comment (nicozar95) [inline config/database.yml]: Also remember to add a default name by doing this: `<%= ENV.fetch('VARIABLE', 'wbooks-api') %>` if there is no `VARIABLE` set on the `.env` file it should use the value `wbooks-api`

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
+#
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
+  <<: *default
+  database: wbooks-api_production
```
Comment (Gmarn) [inline config/database.yml:85]: Ok surely to deploy you will need to use databas_url but when this time came we will see it ;)

**Review by Gmarn — APPROVED** [review]

**Review by nicozar95 — APPROVED** [review]

**Pull request merged**